graphql-server/go-server/pkg/worker/worker.go

```diff
@@ -91,6 +97,7 @@ func worker2model(ctx context.Context, c dynamic.Interface, obj *unstructured.Un
 		UpdateTimestamp:   &updateTime,
 		Resources:         resources,
 		ModelTypes:        "unknown",
+		API:               &gateway.APIServer,
```

Comment:
Should we add a `external apiserver`  in our configmap?  And use the external by default at this place